ReactiveAnimation/Constraint.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Constraint.swift
+//  ReactiveAnimation
+//
+//  Created by Indragie on 3/25/15.
+//  Copyright (c) 2015 ReactiveCocoa. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+
+// Wraps an NSLayoutConstraint for animating its constant using signals.
+public struct RANConstraint {
```

Comment:
Should this be named `Constraint` instead? I named it this way to be consistent with `RANView`.